tests/vmi_lifecycle_test.go

```diff
@@ -1570,7 +1570,7 @@ var _ = Describe("[rfe_id:273][crit:high][arm64][vendor:cnv-qe@redhat.com][level
 				Entry("[test_id:1654]with default grace period seconds", int64(-1)),
 			)
 		})
-		Context("with grace period greater than 0", func() {
+		Context("with grace period greater than 0", decorators.WgS390x, func() {
```

Comment:
Why do we need this special casing for S390x? I'd just revert the changes on this file.

Reply:
As per my understing , right now we are running s390x  tests in post merge and which  are "decorators.WgS390x"

Reply:
ok gotcha.

Reply:
ya .. you are right , my bad